mycode/iteration2.sql

```diff
@@ -0,0 +1,262 @@
+-- MySQL Workbench Forward Engineering
```

Comment:
Casi, todos los interfaces gráficos de gestión de bases de datos te permiten construir la base de datos física a partir del diagrama de relaciones. Así, cogeríamos el código SQL que te da workbench de tu diagrama y lo ejecutaríamos en las tabs de queries. En tu caso, tu código da error en la creación de la primera tabla que es la de usuariosadicional. Esto hace que no te cree las tablas. Esto ocurre porque estamos intentando forzar la relación one-to-one y en el "mundo real" puede que no tenga demasiado sentido. Te adjunto pantallazo de la ejecución del código de esta ejecución. 
<img width="1440" alt="screen shot 2018-10-22 at 08 39 49" src="https://user-images.githubusercontent.com/22130700/47284956-7e6d7900-d5e9-11e8-8005-bff58ae3cde6.png">

No puedo comentar en fichero de imagen por lo que te dejo aquí los comentarios. El diagrama de relaciones tiene buena pinta, tiene su lógica intrínseca. Las tablas de abajo, facturación, y usuarios adic podrían dar problemas pero habría que comprobarlo. El código sql creo que no corresponde a esta iteración, la última con facturación por lo que no puedo comprobarlo.